src/picom.c

```diff
@@ -2597,12 +2598,31 @@ void set_rr_scheduling(void) {
 	param.sched_priority = priority;
 	ret = sched_setscheduler(0, SCHED_RR, &param);
 	if (ret != 0) {
-		log_info("Failed to set real-time scheduling priority to %d. Consider "
-		         "giving picom the CAP_SYS_NICE capability",
-		         priority);
+		log_info("Failed to set real-time scheduling priority to %d.", priority);
 		return;
 	}
 	log_info("Set real-time scheduling priority to %d", priority);
+#else        // OpenBSD real-time scheduling support
```

Comment:
you are duplicating more code than is necessary. also I think you need to add [`dependency('threads')`](https://mesonbuild.com/Threads.html)

Reply:
You are right, I fix this and dedup code too